sox.features.js

```diff
@@ -468,27 +468,10 @@
     shareLinksPrivacy: function() {
       // Description: Remove your user ID from the 'share' link
 
-      const questionTarget = document.getElementById('question');
-      const answersTargets = document.getElementsByClassName('answer');
-      sox.helpers.observe([questionTarget, ...answersTargets], '.share-tip', () => {
-        const toRemove = ' (includes your user id)';
-        const popup = document.getElementsByClassName('share-tip')[0];
-        if (!popup) return;
-
-        // Do nothing if the function's already done its thing
-        const popupTextInfo = popup.querySelector('.js-text');
-        const origText = popupTextInfo.innerText;
-        if (origText.indexOf(toRemove) == -1) return;
-
-        const input = popup.querySelector('input');
-        const origLink = input.value;
-        const newLink = origLink.match(/.+\/(q|a)\/[0-9]+/g)[0];
-
-        popupTextInfo.innerText = origText.replace(toRemove, '');
-
-        input.value = newLink;
-        input.select();
-      });
+      $('.js-subtitle').remove(); // Remove the 'includes your user id' string
```

Comment:
I think it's possible SE will end up using the `js-subtitle` class for other elements soon (and not just for the 'includes your user id' text), so could you make this selector a bit more specific?

I think it should be fairly safe for us to just do `.post-menu .js-subtitle` instead.